templates/apim/apim-group.json

```
@@ -18,17 +20,19 @@
       "type": "string"
     }
   },
-  "variables": {},
+  "variables": {
+    "isExternal": "[and(greater(length(parameters('tenantId')) ,0), greater(length(parameters('aadGroupObjectId')), 0))]"
+  },
   "resources": [
     {
-      "name": "[concat(parameters('apimName'), '/', parameters('aadGroupObjectId'))]",
+      "name": "[concat(parameters('apimName'), '/', if(variables('isExternal'), parameters('aadGroupObjectId'), parameters('groupDisplayName')))]",
```

```suggestion
      "name": "[concat(parameters('apimName'), '/', if(variables('isExternal'), parameters('groupDisplayName'), parameters('aadGroupObjectId')))]",
```
Should the order for the if statement be the other way round?